tests/xharness/Execution/ProcessManager.cs

```diff
@@ -14,6 +14,9 @@
 
 namespace Xharness.Execution {
 	public class ProcessManager : IProcessManager {
+		const string xcodeDefaultPath = "/Applications/Xcode.app/";
```

Comment:
IMO it makes more sense to call `xcode-select -p`

Reply:
Correct, and you are making my life harder ;)

Reply:
🔨 
I care more about your future, than present, self ;-)